dev/sparktestsupport/modules.py

```diff
@@ -469,7 +469,7 @@ def __hash__(self):
     name="yarn",
     dependencies=[],
     source_file_regexes=[
-        "yarn/",
+        "resource-managers/yarn/",
```

Comment:
Oh. Nevermind, I thought those were file paths like these. So fine the way they are.